frontends/web/.gitignore

```diff
@@ -0,0 +1,31 @@
+/.eslintcache
+/build/
+/coverage/
+/node_modules/
+**/*/output.log
+
+## Generated by 'typings-for-css-modules-loader': ##
+/src/**/*.css.d.ts
+/yarn-error.log
+/size-plugin.json
+
+
+# Ignore unfinished locales
+/src/locales/
+# Except the following:
+!/src/locales/en/
+!/src/locales/de/
+!/src/locales/fr/
+!/src/locales/ja/
+!/src/locales/ru/
+!/src/locales/ms/
+!/src/locales/pt/
+!/src/locales/hi/
+!/src/locales/bg/
+!/src/locales/tr/
+!/src/locales/zh/
+!/src/locales/fa/
+!/src/locales/es/
+!/src/locales/sl/
+!/src/locales/he/
+!/src/locales/it/
```

Comment:
I also have problem that VS Code couldn't find en/app.json becuase it is ignored...

could you change it to https://github.com/thisconnect/bitbox-wallet-app/commit/a9542169804df3d64351214433fad77f7ad2ee67

Reply:
The issue I had was that VS Code couldn't `en/app.json`, because `src/locales` is ignored but then each lang is un-ignored again.

I left a comment in my commit `# /frontends/web/src/locales/<unfinished>`, where `<unfinished>` is just a placeholder to replace once we have a new language that is not ready... 
In fact NL is coming soon, so once we have dutch we might have to ignore it until it is ready.